src/core/raster/qgsrasterdrawer.h

```diff
@@ -54,6 +61,15 @@ class CORE_EXPORT QgsRasterDrawer
      */
     void draw( QPainter *p, QgsRasterViewPort *viewPort, const QgsMapToPixel *qgsMapToPixel, QgsRasterBlockFeedback *feedback = nullptr );
```

Comment:
Can we deprecate this one?

Reply:
It's used in other places where it seems it's useful to have that version of draw (e.g. QgsRasterLayer::draw), so maybe we can leave it as is?